blocks/editable/index.js

```diff
@@ -709,6 +709,7 @@ export default class Editable extends Component {
 						{ MultilineTag ? <MultilineTag>{ placeholder }</MultilineTag> : placeholder }
 					</Tagname>
 				}
+				{ focus && <Slot name="Formatting.LinkDialog" /> }
```

Comment:
Should we rename the slot. I think it's meaning should be something related to its position and not what's inside it. Maybe something like `Editable` or  `Editable.Siblings`